### PR TITLE
DOCSUP-7322: Edit and translate to Russian — Cell Towers dataset 

### PR DESCRIPTION
**Pull request by AnaUvarova**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #21278 and #21266.

### CODE REVIEW

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+10 rows in set. Elapsed: 0.019 sec. Processed 43.28 million rows, 86.55 MB (2.33 billion rows/s., 4.65 GB/s.)
+```
+
+Здесь можно заглянуть в словарь: [термин "мобильный код страны"](https://ru.wikipedia.org/wiki/Mobile_Country_Code).
+
+В итоге получаем самые популярные страны: США, Германия, Россия.
+
+Также вам может понадобится создать [внешний словарь](../../sql-reference/dictionaries/external-dictionaries/external-dicts.md) в ClickHouse для того, чтобы объяснить эти значения.
+
+### Пример применения функции `pointInPolygon` {#example-of-using}
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:102]:
```suggestion
## Пример использования {#use-case}
Рассмотрим применение функции `pointInPolygon`.
```

File: docs/en/getting-started/example-datasets/cell-towers.md [hunk 2]
```diff
@@ -55,10 +55,9 @@ Insert the dataset:
 clickhouse-client --query "INSERT INTO cell_towers FORMAT CSVWithNames" < cell_towers.csv
 ```
 
+## Run some queries {#run-some-queries}
```
Comment (adevyatova) [inline docs/en/getting-started/example-datasets/cell-towers.md:58]:
```suggestion
## Examples {#examples}
```

File: docs/en/getting-started/example-datasets/cell-towers.md [hunk 3]
```diff
@@ -55,10 +55,9 @@ Insert the dataset:
...
 
-## Run some queries
-
-Number of cell towers by type:
+A number of cell towers by type:
```
Comment (adevyatova) [inline docs/en/getting-started/example-datasets/cell-towers.md:60]:
```suggestion
1. A number of cell towers by type:
```

File: docs/en/getting-started/example-datasets/cell-towers.md [hunk 4]
```diff
@@ -95,12 +94,12 @@ SELECT mcc, count() FROM cell_towers GROUP BY mcc ORDER BY count() DESC LIMIT 10
...
 
-So, the top countries are USA, Germany and Russia.
+So, the top countries are: the USA, Germany, and Russia.
 
-You may want to create an [External Dictionary](../../sql-reference/dictionaries/external-dictionaries/external-dicts/) in ClickHouse to decode these values.
+You may want to create an [External Dictionary](../../sql-reference/dictionaries/external-dictionaries/external-dicts.md) in ClickHouse to decode these values.
 
 
-### Example of using `pointInPolygon` function
+### Example of using `pointInPolygon` function {#example-of-using}
```
Comment (adevyatova) [inline docs/en/getting-started/example-datasets/cell-towers.md:102]:
```suggestion
## Use-case {#use-case}
Using `pointInPolygon` function.
```

File: docs/en/sql-reference/statements/alter/ttl.md [hunk 5]
```diff
@@ -82,4 +82,4 @@ The `TTL` is no longer there, so the second row is not deleted:
 ### See Also
```
Comment (adevyatova) [inline docs/en/sql-reference/statements/alter/ttl.md:82]:
```suggestion
**See Also**
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+│ LTE   │ 12101148 │
+│ GSM   │  9931312 │
+│ CDMA  │   556344 │
+│ NR    │      867 │
+└───────┴──────────┘
+
+5 rows in set. Elapsed: 0.011 sec. Processed 43.28 million rows, 43.28 MB (3.83 billion rows/s., 3.83 GB/s.)
+```
+
+Количество вышек по мобильному коду страны (MCC):
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:75]:
```suggestion
2. Количество вышек по [мобильному коду страны (MCC)](https://ru.wikipedia.org/wiki/Mobile_Country_Code):
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+```
+
+5. Добавьте датасет:
+```
+clickhouse-client --query "INSERT INTO cell_towers FORMAT CSVWithNames" < cell_towers.csv
+```
+
+## Как проверить некоторые запросы {#run-some-queries}
+
+Количество вышек по типам:
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:59]:
```suggestion
1. Количество вышек по типам:
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+│ 234 │ 1618924 │
+│ 510 │ 1353998 │
+│ 440 │ 1343355 │
+│ 311 │ 1332798 │
+└─────┴─────────┘
+
+10 rows in set. Elapsed: 0.019 sec. Processed 43.28 million rows, 86.55 MB (2.33 billion rows/s., 4.65 GB/s.)
+```
+
+Здесь можно заглянуть в словарь: [термин "мобильный код страны"](https://ru.wikipedia.org/wiki/Mobile_Country_Code).
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:96]:
```suggestion
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+
+В итоге получаем самые популярные страны: США, Германия, Россия.
+
+Также вам может понадобится создать [внешний словарь](../../sql-reference/dictionaries/external-dictionaries/external-dicts.md) в ClickHouse для того, чтобы объяснить эти значения.
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:100]:
```suggestion
Вы также можете создать [внешний словарь](../../sql-reference/dictionaries/external-dictionaries/external-dicts.md) в ClickHouse для того, чтобы расшифровать эти значения.
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+│ 440 │ 1343355 │
+│ 311 │ 1332798 │
+└─────┴─────────┘
+
+10 rows in set. Elapsed: 0.019 sec. Processed 43.28 million rows, 86.55 MB (2.33 billion rows/s., 4.65 GB/s.)
+```
+
+Здесь можно заглянуть в словарь: [термин "мобильный код страны"](https://ru.wikipedia.org/wiki/Mobile_Country_Code).
+
+В итоге получаем самые популярные страны: США, Германия, Россия.
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:98]:
```suggestion
Можно увидеть, что по количеству вышек лидируют следующие страны: США, Германия, Россия.
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+SELECT count() FROM cell_towers WHERE pointInPolygon((lon, lat), (SELECT * FROM moscow))
+
+┌─count()─┐
+│  310463 │
+└─────────┘
+
+1 rows in set. Elapsed: 0.067 sec. Processed 43.28 million rows, 692.42 MB (645.83 million rows/s., 10.33 GB/s.)
+```
+
+Можно попробовать другие запросы с помощью интерактивного ресурса [Песочницы](https://gh-api.clickhouse.tech/play?user=play), например, [вот так](https://gh-api.clickhouse.tech/play?user=play#U0VMRUNUIG1jYywgY291bnQoKSBGUk9NIGNlbGxfdG93ZXJzIEdST1VQIEJZIG1jYyBPUkRFUiBCWSBjb3VudCgpIERFU0M=). Но тут нельзя создавать временные таблицы.
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:128]:
```suggestion
Вы можете протестировать другие запросы с помощью интерактивного ресурса [Песочницы](https://gh-api.clickhouse.tech/play?user=play). Например, [вот так](https://gh-api.clickhouse.tech/play?user=play#U0VMRUNUIG1jYywgY291bnQoKSBGUk9NIGNlbGxfdG93ZXJzIEdST1VQIEJZIG1jYyBPUkRFUiBCWSBjb3VudCgpIERFU0M=). Однако, обратите внимание, что здесь нельзя создавать временные таблицы.
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
+---
+toc_priority: 21
+toc_title: Вышки сотовой связи
+---
+
+# Вышки сотовой связи {#cell-towers}
+
+Источник этого датасета - самая большая в мире открытая база данных о сотовых вышках - [OpenCellid](https://www.opencellid.org/). К 2021-му году здесь накопилось более, чем 40 миллионов записей о сотовых вышках (GSM, LTE, UMTS, и т.д.) по всему миру с их географическими координатами и метаданными (код страны, сети, и т.д.).
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:8]:
```suggestion
Источник этого набора данных (dataset) - самая большая в мире открытая база данных о сотовых вышках - [OpenCellid](https://www.opencellid.org/). К 2021-му году здесь накопилось более, чем 40 миллионов записей о сотовых вышках (GSM, LTE, UMTS, и т.д.) по всему миру с их географическими координатами и метаданными (код страны, сети, и т.д.).
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+
+OpenCelliD Project имеет лицензию Creative Commons Attribution-ShareAlike 4.0 International License, и мы перераспределяем снэпшот датасета по условиям этой же лицензии. После авторизации можно загрузить последнюю версию датасета.
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:10]:
```suggestion
OpenCelliD Project имеет лицензию Creative Commons Attribution-ShareAlike 4.0 International License, и мы распространяем снэпшот набора данных по условиям этой же лицензии. После авторизации можно загрузить последнюю версию датасета.
```

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+)
+ENGINE = MergeTree ORDER BY (radio, mcc, net, created);
+```
+
+5. Добавьте датасет:
+```
+clickhouse-client --query "INSERT INTO cell_towers FORMAT CSVWithNames" < cell_towers.csv
+```
+
+## Как проверить некоторые запросы {#run-some-queries}
```
Comment (adevyatova) [inline docs/ru/getting-started/example-datasets/cell-towers.md:57]:
```suggestion
## Примеры {#examples}
```

File: docs/en/sql-reference/statements/alter/ttl.md (hunk 5 above)
Comment (alexey-milovidov) [reply docs/en/sql-reference/statements/alter/ttl.md:82]: Why need these meaningless changes?

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+
+Источник этого датасета - самая большая в мире открытая база данных о сотовых вышках - [OpenCellid](https://www.opencellid.org/). К 2021-му году здесь накопилось более, чем 40 миллионов записей о сотовых вышках (GSM, LTE, UMTS, и т.д.) по всему миру с их географическими координатами и метаданными (код страны, сети, и т.д.).
+
+OpenCelliD Project имеет лицензию Creative Commons Attribution-ShareAlike 4.0 International License, и мы перераспределяем снэпшот датасета по условиям этой же лицензии. После авторизации можно загрузить последнюю версию датасета.
+
+## Как получить датасет {#get-the-dataset}
+
+1. Загрузите снэпшот датасета за февраль 2021 [отсюда](https://datasets.clickhouse.tech/cell_towers.csv.xz) (729 MB).
+
+2. Если нужно, проверьте полноту при помощи команды:
```
Comment (alexey-milovidov) [inline docs/ru/getting-started/example-datasets/cell-towers.md:16]: Целостность.

md5 проверяет не только полноту, но и целостность.

File: docs/ru/getting-started/example-datasets/cell-towers.md [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+    samples UInt32,
+    changeable UInt8,
+    created DateTime,
+    updated DateTime,
+    averageSignal UInt8
+)
+ENGINE = MergeTree ORDER BY (radio, mcc, net, created);
+```
+
+5. Добавьте датасет:
```
Comment (alexey-milovidov) [inline docs/ru/getting-started/example-datasets/cell-towers.md:52]: Вставьте данные.

File: docs/en/sql-reference/statements/alter/ttl.md (hunk 5 above)
Comment (adevyatova) [reply docs/en/sql-reference/statements/alter/ttl.md:82]: Specifically, this is not necessary. In some places, there are anchors for "see also" - they get in the way. We specified this in the templates.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**